src/Main.java

```diff
@@ -1,5 +1,58 @@
+import java.util.Arrays;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        System.out.println("\nЗадача 1");
+        System.out.println();
+        int[] weight = new int[3];
+        weight[0] = 1;
+        weight[1] = 2;
+        weight[2] = 3;
+        System.out.println(weight[0]);
+        System.out.println(weight[1]);
+        System.out.println(weight[2]);
+        System.out.println();
+        double[] first = {1.57, 7.654, 9.986};
+        System.out.println(first[0]);
+        System.out.println(first[1]);
+        System.out.println(first[2]);
+        System.out.println();
+        int[] array = {12, 13, 11, 99};
+        System.out.println(array[0]);
+        System.out.println(array[1]);
+        System.out.println(array[2]);
+        System.out.println(array[3]);
+        System.out.println();
+
+        System.out.println("\nЗадача 2");
+        for (int i = 0; i < weight.length; i++) {
+            System.out.print(weight[i] + ", ");
+        }
+        for (double i = 0; i < first.length; i++) {
```

Comment:
счетчик - натуральные числа. double это вещественное
```suggestion
        for (int i = 0; i < first.length; i++) {
```